pkg/controller/job/success_policy_test.go

```diff
@@ -353,6 +391,7 @@ func TestMatchSuccessPolicy(t *testing.T) {
 	}
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
+			featuregatetesting.SetFeatureGateDuringTest(t, feature.DefaultFeatureGate, features.JobSuccessPolicy, tc.enableJobSuccessPolicy)
```

Comment:
Oops, I forgot to reset the feature gates here.
I try to fix this issue in the master branch.

/hold

Reply:
uhm, it seems that this is limitations only in the 1.30 or before branches.

Reply:
/hold cansel

Reply:
/hold cancel